src/applications/pensions/components/HomeAcreageValueReview.jsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { isValidCurrency } from '../validation';
+import { isHomeAcreageMoreThanTwo, formatCurrency } from '../helpers';
+
+export const content = {
+  label: 'What’s the value of the land that’s more than 2 acres?',
+  error: 'Please enter a valid dollar amount',
+  title: 'Home acreage value',
+  edit: 'Edit',
+};
+
+const HomeAcreageValueReview = ({ data, editPage }) => {
+  const { homeAcreageValue = '' } = data;
+  const error =
+    homeAcreageValue !== '' && !isValidCurrency(homeAcreageValue) ? (
+      <strong className="usa-input-error-message">{content.error}</strong>
+    ) : null;
+  const labelWrapClasses = error
+    ? 'vads-u-border-left--4px vads-u-border-color--secondary-dark vads-u-padding-left--1p5'
+    : '';
+
+  return isHomeAcreageMoreThanTwo(data) ? (
+    <div className="form-review-panel-page pensions-custom-review">
+      <div className="form-review-panel-page-header-row">
+        <h4 className="form-review-panel-page-header vads-u-font-size--h5 vads-u-margin--0">
+          {content.title}
+        </h4>
+        <va-button
+          secondary
+          class="edit-page float-right"
```

Comment:
Instead of using these classes, we can just use the flexbox alignment utility class since the button is inside of a flexbox
```suggestion
          class="vads-u-justify-content--flex-end"
```